src/M6Web/Bundle/ElasticsearchBundle/Connection/CurlMultiConnectionDecorator.php

```diff
@@ -45,11 +45,14 @@ public function performRequest($method, $uri, $params = null, $body = null, $opt
             throw $e;
         }
 
+        $took = $this->extractTookFromResponse($response);
+
         $this->dispatchEvent(
             $method,
             $uri,
-            $response['requestInfo']['http_code'],
-            $response['requestInfo']['total_time'] * 1000  // Convert from seconds to milliseconds
+            $response['status'],
```

Comment:
Ok, I really thought I tested this with the `CurlMultiConnectionDecorator` but it seems that this is not the case :smile: 

:+1: for this modification.